snat.sh

```diff
@@ -5,13 +5,16 @@ while ! ip link show dev eth1; do
   sleep 1
 done
 
+sysctl -q -w net.ipv4.conf.all.rp_filter=0
+sysctl -q -w net.ipv4.conf.eth0.rp_filter=0
+sysctl -q -w net.ipv4.conf.eth1.rp_filter=0
+sysctl -q -w net.ipv4.conf.default.rp_filter=0
+
+
 # enable IP forwarding and NAT
 sysctl -q -w net.ipv4.ip_forward=1
 sysctl -q -w net.ipv4.conf.eth1.send_redirects=0
-iptables -t nat -A POSTROUTING -o eth1 -j MASQUERADE
-
-# switch the default route to eth1
-ip route del default dev eth0
+iptables -t nat -A POSTROUTING -o eth0 -j MASQUERADE
```

Comment:
I think this breaks the fixed source IP feature. ENI of eth0 will be changed when the instance is recreated by Auto Scaling Group.

Reply:
will this be merged any time soon?